### ISSUE TITLE
conda-forge package?

### ISSUE THREAD

**Issue by ivirshup** [scverse/rapids_singlecell#65]
Could this be made available through conda-forge? It would be nice to be able to set up an environment with this package in a single command. I'm a little unclear on how this works given the nvidia dependencies.

**Comment by Intron7** [comment]
I would love that conda-forge release as well. As far as I understand it, to get a package into forge all deps need to be in forge. Since rapids and the cuda dependencies are in the rapids and nvidia channels respectively I don't now how to do it.

**Comment by Intron7** [comment]
yeah cupy is sadly the only thing that's currently on forge. If somebody puts rapids on there I would add rapids-singlecell immediately.

I think a bar bones version like I have with pypi defeats the purpose of having it on conda. I don't know if they even allow this.
